config/routes/publish.rb

```diff
@@ -23,8 +23,12 @@
 get '/privacy', to: 'pages#privacy', as: :privacy
 get '/terms-conditions', to: 'pages#terms', as: :terms
 get '/how-to-use-this-service', to: 'pages#how_to_use_this_service'
+get '/how-to-use-this-service/add-an-organisation', to: 'pages#add_an_organisation', as: :add_an_organisation
```

Comment:
Could we use a `namespace` or `scope` all the routes within `how-to-use-this-service`? eg:

```ruby
scope path: "how-to-use-this-service" do
 get "/add-an-organisation"
 etc
end
```

Reply:
Done https://github.com/DFE-Digital/publish-teacher-training/pull/3491/commits/69337e8711a54c4a4d464194bddcc496e4984e6f